app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="lg:h-24 h-16 bg-black sticky px-4 lg:px-20 flex justify-between items-center">
+  <a href=<%= root_path %>>
```

Comment:
```suggestion
  <a href="<%= root_path %>">
```

minor comment, but I think using quotes is a best practice, it can help with the different values that can be returned by ruby (e.g. empty, with spaces, dashes, etc